### PR TITLE
Add Marcin Zabłocki to Kedro TSC

### PR DESCRIPTION
**Pull request by marrrcin**
## Description
I apply for the TSC membership in the great Kedro community :) 

## Checklist

- [x] Read the [contributing](https://github.com/kedro-org/kedro/blob/main/CONTRIBUTING.md) guidelines
- [x] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [x] Updated the documentation to reflect the code changes
- [x] ~~Added a description of this change in the [`RELEASE.md`](https://github.com/kedro-org/kedro/blob/main/RELEASE.md) file~~ -- N/A
- [x] ~~Added tests to cover my changes~~ -- N/A

### CODE REVIEW

File: docs/source/contribution/technical_steering_committee.md [hunk 1]
```diff
@@ -74,7 +74,8 @@ Currently, the core Kedro team consists of:
 [Rashida Kanchwala](https://github.com/rashidakanchwala),
 [Sajid Alam](https://github.com/SajidAlamQB),
 [Stephanie Kaiser](https://github.com/stephkaiser),
-[Tynan DeBold](https://github.com/tynandebold) and
+[Tynan DeBold](https://github.com/tynandebold),
+[Marcin Zabłocki](https://github.com/marrrcin) and
```
Comment (merelcht) [inline docs/source/contribution/technical_steering_committee.md:78]: @marrrcin This list is alphabetical by first name, so you'll have to move yourself up a bit 🙂

**Review by astrojuanlu — APPROVED** [review]
Welcome to the team @marrrcin! 👏🏽

**Review by stichbury — APPROVED** [review]
Pleased to see you here @marrrcin 💃

**Comment by noklam** [comment]
Welcome @marrrcin !

**Review by datajoely — APPROVED** [review]

**Comment by merelcht** [comment]
We have a majority 👍 vote, so this can now be merged 🎉 

@marrrcin the DCO check is failing because your commits haven't been signed-off. You can fix this by following these instructions: https://github.com/kedro-org/kedro/pull/2513/checks?check_run_id=12717495655

**Comment by marrrcin** [comment]
@merelcht fixed, ready to merge

**Pull request merged**